types/index.js

```diff
@@ -13,5 +13,6 @@ module.exports = [
   require('./unionpay'),
   require('./troy'),
   require('./elo'),
-  require('./uatp')
+  require('./uatp'),
```

Comment:
No, these cards are excluded from the default list intentionally

https://github.com/bendrucker/creditcards-types#co-branded-cards